### ISSUE TITLE
OSSI page alignment

### ISSUE THREAD

**Issue by krokicki** [JaneliaSciComp/ossi-website#255]
The alignment of the text on the OSSI page is kind of squished to the left. Could the text area be centered and a bit wider to make it easier to read?

![Screenshot 2024-05-28 at 9 42 41 AM](https://github.com/JaneliaSciComp/ossi-website/assets/607324/95a832fe-d7d8-428d-8f05-e767b938438e)

**Comment by allison-truhlar** [comment]
@krokicki I increased the left padding and the text size - before: 
![before](https://github.com/JaneliaSciComp/ossi-website/assets/114596142/48ea3e08-6314-486a-81db-636119855129)
and after (on branch ossi-page-alignment):
![OSSI-after](https://github.com/JaneliaSciComp/ossi-website/assets/114596142/22b7da45-becf-49a5-bc12-74f643b24513)

I didn't want to expand the column width too much, to keep the number of characters per line readable, and I didn't want to add too much extra padding, to keep the look of this page aligned with the others. Let me know if you'd prefer further tweaks, though!

**Comment by krokicki** [comment]
That's much better!

A lot of sites will center the text content column on the page, because on a very large screens it's a little awkward to have all the content to one side. Basically what you did on the blog pages, so as you make the window wider the content stays in the middle for easy reading. But maybe that's a preference, and I think it's okay now if you prefer it styled this way.

**Comment by allison-truhlar** [comment]
@krokicki I think I've adjusted the styling now to address both of our thinking/preferences. Now, all the content, including the hero, is left-aligned but overall centered in the page on the homepage and the OSSI page. I kept the hero left-aligned on the Projects and Ecosystems search pages, and it aligns with the left side of the content on those pages (which fills the page better due to the card grid). 

This video attempts to demo what I'm getting at above: https://www.awesomescreenshot.com/video/28109629?key=72b131b4e305f09556007dc922868251
